minifi/minifi-commons/minifi-commons-api/src/main/java/org/apache/nifi/minifi/commons/api/MiNiFiProperties.java

```diff
@@ -91,6 +91,7 @@ public enum MiNiFiProperties {
     C2_SECURITY_KEYSTORE_PASSWORD("c2.security.keystore.password", "", true, false, VALID),
     C2_SECURITY_KEYSTORE_TYPE("c2.security.keystore.type", "JKS", false, false, VALID),
     C2_REQUEST_COMPRESSION("c2.request.compression", "none", false, true, VALID),
+    C2_BOOTSTRAP_ACKNOWLEDGE_TIMEOUT("c2.bootstrap.acknowledge.timeout", "60 sec", false, true, VALID),
```

Comment:
This seems like a lot for a default if I understand the use case right but maybe there are arguments against lowering this number. Just wanted to raise I thought 10-15sec should be sufficient or for complex flows restart could take really long?

Reply:
Lowered to 15 secs